src/data/pokemon/item_effects.h

```diff
@@ -113,6 +113,41 @@ const u8 gItemEffect_RevivalHerb[10] = {
     [9] = -20, // Friendship change, high
 };
 
+const u8 gItemEffect_Remedy[10] = {
+    [4] = ITEM4_HEAL_HP,
+    [5] = ITEM5_FRIENDSHIP_ALL,
+    [6] = 20, // Amount of HP to recover
+    [7] = -5, // Friendship change, low
+    [8] = -5, // Friendship change, mid
+    [9] = -10, // Friendship change, high
+};
+
+const u8 gItemEffect_FineRemedy[10] = {
+    [4] = ITEM4_HEAL_HP,
+    [5] = ITEM5_FRIENDSHIP_ALL,
+#if I_HEALTH_RECOVERY >= GEN_7
```

Comment:
Ternaries

Reply:
That's already the case though? 🤔

Reply:
Oh, they got converted to ternaries at some point after I started this branch, will take care of it!